### PR TITLE
Create cask for 'ungoogled-chromium'

### PR DESCRIPTION
**Pull request by Git-Jiro**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [ ] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/ungoogled-chromium.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'ungoogled-chromium' do
+  version '53.0.2785.116-1.2'
+  sha256 '1ab010b83779f50b69c56a72659437c50ab1f1fc73b9870a77ba3063ee0ef5a5'
+
+  url 'https://github.com/Eloston/ungoogled-chromium/releases/download/53.0.2785.116-1.2/ungoogled-chromium_53.0.2785.116-1.2_macos.dmg'
```
Comment (reitermarkus) [inline Casks/ungoogled-chromium.rb]: This needs an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by reitermarkus — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
As per the token reference, this should be `eloston-chromium`.

> Details of software names and brands will inevitably be lost in the conversion to a minimal token. To capture the vendor’s full name for a distribution, use the [`name`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/name.md) within a Cask. `name` accepts an unrestricted UTF-8 string.

Though on this particular case, I was a bit ambivalente if it wasn’t worthy of an exception. I leaned towards “no”, though. This had a (somewhat) meteoric rise, but I wonder for its future. As I’ve seen pointed out, trading known evil (Google) for binaries made by an unknown third party is a dubious move. It’s not a bad idea, though, and it’s conceivable other projects like this might pop up in the future. Rather than treat “Ungoogled Chromium” as something special due to a catchy name (not saying this in a bad way), we should treat this as just another fork, just like [`freesmug-chromium`](https://github.com/caskroom/homebrew-cask/blob/master/Casks/freesmug-chromium.rb), because in the end, that’s what it is.

File: Casks/eloston-chromium.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+cask 'eloston-chromium' do
   version '53.0.2785.116-1.2'
   sha256 '1ab010b83779f50b69c56a72659437c50ab1f1fc73b9870a77ba3063ee0ef5a5'
 
   url "https://github.com/Eloston/ungoogled-chromium/releases/download/#{version}/ungoogled-chromium_#{version}_macos.dmg"
   appcast 'https://github.com/Eloston/ungoogled-chromium/releases.atom',
-    checkpoint: 'fb8dc0c21f90efbde4e82a23c76fe559991632f0b4bcd551e11a99976be84f34'
-  name 'Ungoogled Chromium'
+          checkpoint: 'fb8dc0c21f90efbde4e82a23c76fe559991632f0b4bcd551e11a99976be84f34'
+  name 'Ungoogled Chromium by Eloston'
```
Comment (vitorgalvao) [inline Casks/eloston-chromium.rb]: No need for the `by Eloston`. This is a place for the name the developer gives to the app, however they choose to name it.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by adidalal — APPROVED** [review]

**Comment by chovy** [comment]
Not sure what I'm doing wrong.

```
$ brew search ungoogled-chromium
No formula found for "ungoogled-chromium".
==> Searching pull requests...
```

**Comment by reitermarkus** [comment]
> Rather than treat “Ungoogled Chromium” as something special due to a catchy name (not saying this in a bad way), we should treat this as just another fork, just like freesmug-chromium, because in the end, that’s what it is.

It was added as `eloston-chromium`.

**Pull request merged**